src/components/views/settings/SecureBackupPanel.tsx

```diff
@@ -228,7 +261,7 @@ export default class SecureBackupPanel extends React.PureComponent<{}, IState> {
         } else if (backupInfo) {
             let restoreButtonCaption = _t("Restore from Backup");
 
-            if (MatrixClientPeg.safeGet().getKeyBackupEnabled()) {
+            if (this.state.activeBackupVersion !== null) {
```

Comment:
Did you use `!== null` because of something in the style guide, or are you just confident that `activeBackupVersion` can never be `undefined` and want to express that?

I.e. I'm considering `if (this.state.activeBackupVersion)`.

Reply:
I used `!== null` because I remain confused about the "no implicit casts to boolean" rule in the style guide.

But I'm reasonably sure it can't be `undefined` so the two are hopefully equivalent

Reply:
Right. Seems like something for us to revisit in the weekly meeting. Would you mind raising it?

Reply:
Well, I've opened https://github.com/vector-im/element-web/pull/26220 for now. Let's see where we get to by Thursday :)